client/dashboard/sites/deployments/deployment-logs/deployment-logs-queries.ts

```diff
@@ -0,0 +1,55 @@
+import { queryOptions } from '@tanstack/react-query';
+import wpcom from 'calypso/lib/wp';
+
+export interface LogEntry {
+	message: string;
+	level: string;
+	timestamp: string;
+	context?: {
+		command: {
+			command_identifier: string;
+			exit_code: number;
+		};
+	};
+}
+
+export interface LogEntryDetail {
+	exit_code: number;
+	stdout: Array< string >;
+	stderr: Array< string >;
+}
+
+export const deploymentRunLogsQuery = ( siteId: number, deploymentId: number, runId: number ) =>
```

Comment:
We should move these queries and api calls to `@automattic/api-queries` and `@automattic/api-core`